samples/net/coap_client/src/coap-client.c

```diff
@@ -211,9 +240,29 @@ static void event_iface_up(struct net_mgmt_event_callback *cb,
 
 	net_pkt_frag_add(pkt, frag);
 
-	r = coap_packet_init(&request, pkt, 1, COAP_TYPE_CON,
-			     8, coap_next_token(),
-			     COAP_METHOD_GET, coap_next_id());
+	switch(coap_method)
+	{
+		case 0:
+			r = coap_packet_init(&request, pkt, 1, COAP_TYPE_CON,
+					     8, coap_next_token(),
+					     COAP_METHOD_GET, coap_next_id());
+			break;
+
+		case 1:
+			r = coap_packet_init(&request, pkt, 1, COAP_TYPE_CON,
+					7,My_COAP_token1, COAP_METHOD_PUT, coap_next_id());
```

Comment:
Fix indentation. Also there should be space after ",". You could check these issues before submitting the patch, there is scripts/checkpatch.pl script just for this purpose.